dot/rpc/http.go

```diff
@@ -43,26 +43,23 @@ type HTTPServerConfig struct {
 	SyncStateAPI        SyncStateAPI
 	SyncAPI             SyncAPI
 	NodeStorage         *runtime.NodeStorage
-	RPC                 bool
-	RPCExternal         bool
 	RPCUnsafe           bool
+	RPCExternal         bool
 	RPCUnsafeExternal   bool
 	Host                string
 	RPCPort             uint32
-	WS                  bool
 	WSExternal          bool
-	WSUnsafe            bool
 	WSUnsafeExternal    bool
 	WSPort              uint32
 	Modules             []string
 }
 
 func (h *HTTPServerConfig) rpcUnsafeEnabled() bool {
-	return h.RPCUnsafe || h.RPCUnsafeExternal
+	return h.RPCUnsafeExternal || h.RPCUnsafe
 }
 
 func (h *HTTPServerConfig) wsUnsafeEnabled() bool {
-	return h.WSUnsafe || h.WSUnsafeExternal
```

Comment:
Do we not need WSUnsafe anymore?

Reply:
I believe it was never being used. Please correct me if I'm wrong.